src/stream/src/executor/sink.rs

```diff
@@ -55,17 +55,18 @@ async fn build_sink(
 }
 
 // Drop all the UPDATE/DELETE messages in this chunk.
-fn force_append_only(chunk: StreamChunk, data_types: Vec<DataType>) -> StreamChunk {
+fn force_append_only(chunk: StreamChunk, data_types: Vec<DataType>) -> Option<StreamChunk> {
     let mut builder = DataChunkBuilder::new(data_types, chunk.cardinality() + 1);
     for (op, row_ref) in chunk.rows() {
         if op == Op::Insert {
```

Comment:
So `Update` events are ignored. Previously I thought the `insert` part of `update` would be kept. Did we have any discussion on that?

Reply:
IIUC, keeping `UpdateInsert` means keeping the `UPDATE` operation, which doesn't conform with the semantics of "append-only". cc. @tabVersion @st1page

Reply:
I think eric means we should convert `UpdateInsert` into `Insert` here. I weakly +1 and it makes sense to me.

Reply:
agree with @fuyufjh and @st1page , we map `UpdateInsert` to `Insert` and ignore `UpdateDelete` & `Delete`.

Reply:
Fixed in this PR.